stateMachine/StateMachine.js

```diff
@@ -0,0 +1,85 @@
+const { helpers } = require('brave-alert-lib')
+const stateMachineHelpers = require('./stateMachineHelpers')
+const redis = require('../db/redis')
+const ALERT_REASON = require('../AlertReasonEnum')
+const DOOR_STATUS = require('../SessionStateDoorEnum')
+const STATE = require('./SessionStateEnum')
+
+async function getNextState(location, handleAlert) {
+  try {
+    let state
+    // await redis.beginTransaction()
```

Comment:
I think this comment is here to remind you to put this into a redis transaction.

Reply:
punting on this.